src/OpenLoco/src/Objects/TrainStationObject.h

```diff
@@ -50,10 +50,10 @@ namespace OpenLoco
         uint32_t var_12[4];
         uint8_t numCompatible; // 0x22
         uint8_t mods[7];
-        uint16_t designedYear;                   // 0x2A
-        uint16_t obsoleteYear;                   // 0x2C
-        const std::byte* cargoOffsetBytes[4][4]; // 0x2E
-        const std::byte* manualPower[16];
+        uint16_t designedYear;           // 0x2A
+        uint16_t obsoleteYear;           // 0x2C
+        uint32_t cargoOffsetBytes[4][4]; // 0x2E
+        uint32_t var_6E[16];
```

Comment:
this was called `manualPower` before - should we kepe that name, or was it just a guess at what the variable did?

Reply:
Duncan asked me to change it, apparently it is not used by anything, its also pretty odd that CS left 64 bytes unused like that, according to [this](https://www.tt-wiki.net/wiki/Variable:List#Class_0F_-_Train_Stations) it seems to be related to icons but there are no references in the code to this variable.